005-quarkus-and-maven-profiles/pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-spring-di</artifactId>
-            <version>1.3.2.Final-redhat-00001</version>
+            <version>3.0.0.Alpha3</version>
```

Comment:
module compiles without specifing `3.0.0.Alpha3`, is there advantage in specifing it?

Reply:
There is no 3.based redhat version yet. old versions are pulling in javax stuff
This change is just to make it run ...

Reply:
tbh I don't comprehend. I thought we have beefy scenarios to run against 999-SNAPSHOT and when no version is specified, then `quarkus-spring-di:999-SNAPSHOT` is used and tests passes. please give me a hint that will help me to get it, thanks

Reply:
redhat version is used to ensure RH maven repo got activated ...

There was issue in old Quarkus that this didn't work ... Quarkus 1.3 I guess

Reply:
Maybe I should add a TODO note to move to -redhat when there is 3.x version released

Reply:
ok

Reply:
Added